siliconcompiler/report/streamlit_report.py

```diff
@@ -0,0 +1,113 @@
+import os
+import time
+import tempfile
+import json
+
+try:
+    from streamlit.web import bootstrap
+    from streamlit import config as _config
+    _has_streamlit = True
+except ModuleNotFoundError:
+    _has_streamlit = False
+
+import multiprocessing
+import subprocess
+import atexit
+import shutil
+
+
+class Dashboard():
+    def __init__(self, chip, port=8501):
+        if not _has_streamlit:
+            raise RuntimeError("Streamlit is not permitted on Python 3.6")
```

Comment:
I think this is wrong, I think it's that they moved some things around and we can fix in this PR for 3.6. I'll handle that.